### PR TITLE
copy LICENSE file during autoload dumping ( refs #4288 )

### LINKED ISSUE #4288: composer generated ClassLoader.php references non-existent LICENSE file
After building a composer autoloader, the `vendor/composer/ClassLoader.php` file has the following heading:
```
/*
 * This file is part of Composer.
 *
 * (c) Nils Adermann <naderman@naderman.de>
 *     Jordi Boggiano <j.boggiano@seld.be>
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */
```

but there is no LICENSE distributed. It may make sense to just include a url to the license instead.

### PR DESCRIPTION
**Pull request by alcohol**
closes #4288

### CODE REVIEW

**Comment by Seldaek** [comment]
yay

File: src/Composer/Autoload/AutoloadGenerator.php [hunk 1]
```diff
@@ -750,4 +744,20 @@ protected function sortPackageMap(array $packageMap)
...
         return $sortedPackageMap;
     }
+
+    /**
+     * Copy file using stream_copy_to_stream to work around https://bugs.php.net/bug.php?id=6463
+     *
+     * @param string $source
+     * @param string $target
+     */
+    protected function safeCopy($source, $target)
```
Comment (stof) [inline src/Composer/Autoload/AutoloadGenerator.php]: should be private IMO

Comment (alcohol) [inline src/Composer/Autoload/AutoloadGenerator.php]: all of the helper methods use protected, so I went for consistency :-(

Comment (Seldaek) [inline src/Composer/Autoload/AutoloadGenerator.php]: Oh right, I'll fix that.

Comment (Seldaek) [inline src/Composer/Autoload/AutoloadGenerator.php]: Yeah just saw that, we probably should fix it eventually, but let's 
leave as is for now, I doubt anyone extends this anyway.

**Pull request merged**